src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1172,6 +1172,42 @@ private static string AddToPath(string basePath, string pathToAdd, int insertPos
             return result.ToString();
         }
 
+        /// <summary>
+        /// The available module path types.
+        /// </summary>
+        public enum ModulePathType
```

Comment:
Should this be PSModulePathType to match the method?

Reply:
sure